src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -157,32 +161,36 @@ private function validate(\Twig_Environment $twig, $template, $file)
         return array('template' => $template, 'file' => $file, 'valid' => true);
     }
 
-    private function display(InputInterface $input, OutputInterface $output, $files)
+    private function display(InputInterface $input, OutputInterface $stdout, $output, $files)
```

Comment:
I would rather keep ``$output`` for the object received as ``$output`` in arguments and used a different name for the styling wrapper. It would be more readable IMO